src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -279,6 +279,112 @@
+
+          <div *ngIf="bucketForm.getValue('encryption_enabled')">
+            <div class="form-group row">
+              <div class="cd-col-form-offset">
+                <div class="custom-control custom-radio custom-control-inline pl-5">
+                  <input class="custom-control-input"
+                         formControlName="encryption_type"
+                         id="sse_S3_enabled"
+                         type="radio"
+                         name="encryption_type"
+                         value="AES256"
+                         [attr.disabled]="!s3VaultConfig ? true : null">
+                  <label class="custom-control-label"
+                         for="sse_S3_enabled"
+                         i18n>SSE-S3 Encryption</label>
+                </div>
+              </div>
+            </div>
+
+            <div class="form-group row">
+              <div class="cd-col-form-offset">
+                <div class="custom-control custom-radio custom-control-inline pl-5">
+                  <input class="custom-control-input"
+                         formControlName="encryption_type"
+                         id="kms_enabled"
+                         name="encryption_type"
+                         value="aws:kms"
+                         [attr.disabled]="!kmsVaultConfig ? true : null"
+                         type="radio">
+                  <label class="custom-control-label"
+                         for="kms_enabled"
+                         i18n>Connect to an external key management service</label>
+                </div>
+              </div>
+            </div>
+        
+            <div *ngIf="bucketForm.getValue('encryption_type') == 'aws:kms'">
+              <div class="form-group row">
+                <label class="cd-col-form-label required"
+                      for="kms_provider"
+                      i18n>KMS Provider</label>
+                <div class="cd-col-form-input">
+                  <select id="kms_provider"
+                          name="kms_provider"
+                          class="form-control"
+                          formControlName="kms_provider"
+                          [autofocus]="editing">
+                    <option i18n
+                            *ngIf="kmsProviders !== null"
+                            [ngValue]="null">-- Select a provider --</option>
+                    <option *ngFor="let provider of kmsProviders"
+                            [value]="provider">{{ provider }}</option>
+                  </select>
+                  <span class="invalid-feedback"
+                        *ngIf="bucketForm.showError('kms_provider', frm, 'required')"
+                        i18n>This field is required.</span>
+                </div>
+              </div>
+            </div>
+
+            <div *ngIf="bucketForm.getValue('encryption_type') == 'aws:kms'">
+              <div class="form-group row">
+                <label class="cd-col-form-label"
```

Comment:
I see there's a validation that states this field is required?
```suggestion
                <label class="cd-col-form-label required"
```